### PR TITLE
Feature/print areas

### PR DESCRIPTION
**Pull request by krishnajithr5**

### CODE REVIEW

File: custom-plugins/fictive_studios/admin/migrations/create-tables.php [hunk 1]
```diff
@@ -7,11 +7,13 @@ public function createTables()
     {
         $this->loadMigrations();
         create_templates_tables();
+        create_printing_areas_tables();
     }
 
     public function loadMigrations()
     {
          require_once PLUGIN_ROOT. 'admin/migrations/templates.php';
+         require_once PLUGIN_ROOT. 'admin/migrations/printing-areas.php';
```
Comment (harikrishna266) [inline custom-plugins/fictive_studios/admin/migrations/create-tables.php:16]: Can we use this ? 
plugin_dir_url( dirname( __FILE__ ) )

if it work remove the PLUGIN_ROOT from root of the plugin, we dont't need it.

File: custom-plugins/fictive_studios/admin/printing-area/listing/printing-area-listing.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        $link_url = esc_url(admin_url('admin.php?page=' . PRINTING_AREAS_BUILDER_SLUG));
+    }
+    public function add_submenu()
+    {
+        $submenu = array(
+            'page_title' => __('Printing Areas', 'printing_areas'),
+            'menu_title' => __('Printing Areas', 'printing_areas'),
+            'capability' => 'manage_options',
+            'menu_slug' => 'printing_areas',
```
Comment (harikrishna266) [inline custom-plugins/fictive_studios/admin/printing-area/listing/printing-area-listing.php:16]: this should be not be hardcoded.. should use PRINTING_AREAS_BUILDER_SLUG

File: custom-plugins/fictive_studios/admin/printing-area/listing/printing-area-listing.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+class PrintingAreaListingAdmin
+{
+
+    public function printing_area_admin()
```
Comment (harikrishna266) [inline custom-plugins/fictive_studios/admin/printing-area/listing/printing-area-listing.php:6]: you need to add basic listing...

File: custom-plugins/fictive_studios/admin/migrations/create-tables.php (hunk 1 above)
Comment (krishnajithr5) [reply custom-plugins/fictive_studios/admin/migrations/create-tables.php:16]: I will use `plugin_dir_path(__FILE__)` here

**Review by harikrishna266 — APPROVED** [review]

**Pull request merged**